test/AssemblyUtil_server_test.py

```diff
@@ -82,6 +84,7 @@ def check_fasta_file(self, ws_obj_name, expected_file):
             actual_data = f.read()
         self.assertEqual(actual_data, expected_data)
 
+    @unittest.skip
```

Comment:
Looks like this should be removed

Reply:
This test in the AU_server_test wasn't working, that's why I was skipping it. But Tian fixed the test in this past PR.